.dockerignore

```diff
@@ -0,0 +1,10 @@
+.env
+venv/
+.vscode/
+.git/
+.github/
+.gitignore
+.dockerignore
+README.md
+Dockerfile
+docker-compose.yml

```

Comment:
Why did you add this file to .dockerignore?)

Reply:
I don't know)